docs/index.rst

```diff
@@ -3,6 +3,7 @@
   :hidden:
 
   gcloud-api
+  datastore-api
```

Comment:
If you look at what the document corresponds to, it's just a basic description of the package. (See the screenshots in this PR.)

What should I name the file? What should we put in this files?